db/models/ally.js

```diff
@@ -0,0 +1,29 @@
+'use strict'
+
+const Sequelize = require('sequelize')
+const db = require('APP/db')
+
+const Ally = db.define('allies', {
+  username: Sequelize.STRING,
+  phone_number: {
```

Comment:
True, I just wanted to make sure people were available as needed. But I Guess that concern is better-addressed by the 'urgent' list anyway. I will take this off